### ISSUE TITLE
Add Article for Hello World in Solidity

### ISSUE THREAD

**Issue by jrg94** [jrg94/sample-programs#229]
Thanks, @2Clutch! If you have any resources to help me understand smart contracts and whatnot, let me know.

**Comment by 2Clutch** [comment]
I would recommend,

1. https://bit.ly/2xqZTrr
2. https://bit.ly/2wVwQvl
3. https://bit.ly/2xoOcBe

Hope that helps.

**Comment by jrg94** [comment]
Would you be interested in writing this article? I'm just now experimenting with guest posts (#203), and I figure you're probably more qualified to write this article up. Let me know! 

If so, I'll add you to the site.

**Comment by 2Clutch** [comment]
Yes, most definitely. I will be using the existing articles as a baseline.

**Comment by jrg94** [comment]
Great! Just added you to the site. Thanks for your help.

**Comment by 2Clutch** [comment]
Quick question. Do you have a limit to how long or how short you would like a blog post to be? I started today and it seems my post will be rather lengthy. 

My game plan is start a very technical post and then slowly dumb it down over a couple of revisions. The ultimate objective being anybody reading the post should be able to make sense of it regardless of their prior knowledge.

**Comment by jrg94** [comment]
Typically, my articles in the Hello World Series run about 500 words, but you’re welcome to write as much or as little as you want. Personally, I wouldn’t write much less than 500 words. That said, there’s no upper bound, so feel free to have some fun with it! 

I think your plan sounds great, and I appreciate the help!

**Comment by jrg94** [comment]
Hey! Sorry, I've been on vacation for about a week. I'm digging back into things a bit. Is your article ready for scheduling?

**Comment by 2Clutch** [comment]
I just completed it. Could you take a look and potentially provide some feedback? It's currently saved as a draft.

**Comment by jrg94** [comment]
I took a quick pass of the article. It looks great! I changed a couple headings and added a table of contents as well as the last section.

If you're happy with it, let's find a good featured image. I typically use [Pixabay](https://pixabay.com/). If you find something you like, link it here, and I'll put it in the article. 

Then, I'll schedule the article! 💯

**Comment by jrg94** [comment]
Oh, and don't forget to update your profile! That way you can link to things like your GitHub profile.

**Comment by 2Clutch** [comment]
Just did. Also I've tried uploading the image but it won't load up for some reason. The image is there, but nothing is showing. I left the url in the draft. 

I tried to upload it from my computer and also the web using the url. The image is processing fine, but nothing is showing. I left the url of the image in question in the draft. Would appreciate it if you could take a look.

**Comment by jrg94** [comment]
Yeah, no problem! I think your role on the website won't allow you to add the image yourself. I've added it and scheduled the article for June 20th at 10AM.

**Comment by 2Clutch** [comment]
Blog was officially posted so I am closing the issue.

**Comment by jrg94** [comment]
Thanks for your help! I hope you enjoyed it.

If you're ever interested in writing more or if you'd like to add more code to the repo, let me know.
